.github/workflows/phpunit.yml

```
@@ -81,12 +81,14 @@ jobs:
         working-directory: /tmp/wordpress-${{ env.WP_BRANCH }}/src/wp-content/plugins/jetpack
         run: |
           for PACKAGE in ./packages/*/tests/php; do
-            cd "$PACKAGE/../.."
-            NAME=$(basename $(pwd))
-            echo "::group::Package $NAME"
-            composer phpunit
-            echo "::endgroup::"
-            cd ../..
+            if [ ! -e tests/php/ci-can-run.sh ] || tests/php/ci-can-run.sh; then
+              cd "$PACKAGE/../.."
+              NAME=$(basename $(pwd))
+              echo "::group::Package $NAME"
+              composer phpunit
+              echo "::endgroup::"
+              cd ../..
+            fi
```

You have operations in the wrong order, this winds up looking for ./tests/php/ci-can-run.sh instead of ./packages/$NAME/tests/php/ci-can-run.sh
```suggestion
            cd "$PACKAGE/../.."
            if [ ! -e tests/php/ci-can-run.sh ] || tests/php/ci-can-run.sh; then
              NAME=$(basename $(pwd))
              echo "::group::Package $NAME"
              composer phpunit
              echo "::endgroup::"
            fi
            cd ../..
```
or even
```suggestion
            cd "$PACKAGE/../.."
            NAME=$(basename $(pwd))
            echo "::group::Package $NAME"
            if [ ! -e tests/php/ci-can-run.sh ] || tests/php/ci-can-run.sh; then
              composer phpunit
            fi
            echo "::endgroup::"
            cd ../..
```
depending on how you want it to show up in the logs. Personally I think I'd prefer the latter, but either one is fine with me.